sklearn/model_selection/tests/test_split.py

```diff
@@ -980,6 +980,18 @@ def test_repeated_cv_value_errors():
         assert_raises(ValueError, cv, n_repeats=1.5)
 
 
+def test_repeated_cv_repr():
```

Comment:
Instead of including a for loop internally to the test, we can use pytest to create automatically 2 tests:

```python
@pytest.mark.parametrize(
    "RepeatedCV", [RepeatedKFold, RepeatedStratifiedKFold]
)
def test_repeated_cv_repr(RepeatedCV):
    n_splits, n_repeats = 2, 6
    repeated_cv = RepeatedCV(n_splits=n_splits, n_repeats=n_repeats)
    repeated_cv_repr = ('{}(n_repeats=6, n_splits=2, random_state=None)'
                        .format(repeated_cv.__class__.__name__))
    assert repr(repeated_cv) == repeated_cv_repr
```

I let you make the changes.

Reply:
I've introduced the requested changes. Thanks for the suggestion, it is indeed cleaner and more scalable.